src/Servers/IIS/AspNetCoreModuleV2/DefaultRules.ruleset

```diff
@@ -1,432 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="DefaultRules" ToolsVersion="15.0">
-  <Include Path="allrules.ruleset" Action="Default" />
-  <Rules AnalyzerId="Microsoft.Analyzers.NativeCodeAnalysis" RuleNamespace="Microsoft.Rules.Native">
-    <Rule Id="C26100" Action="Error" />
-    <Rule Id="C26101" Action="Error" />
-    <Rule Id="C26105" Action="Error" />
-    <Rule Id="C26110" Action="Error" />
-    <Rule Id="C26111" Action="Error" />
-    <Rule Id="C26112" Action="Error" />
-    <Rule Id="C26115" Action="Error" />
```

Comment:
How was I seeing these as "CAxxxx"? 😄 

@sharwell These rules are for C++. Should I keep this ruleset as-is?